DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/filter.ts

```diff
@@ -188,6 +195,14 @@ function updateFilterBy(newFilter: string): void {
   element.dispatchEvent(new Event('change'));
 }
 
+function updateFilterCookie(newFilter: string): void {
+  if (newFilter) {
+    Cookies.set(cookieName, newFilter, { expires: cookieExpiry });
+  } else {
+    Cookies.remove(cookieName);

```

Comment:
`updateFilterCookie()` doesn't really say "clear the cookie" to me. I think it would make more sense to have a separate function for each branch:
`updateFilterCookie[Value?](newFilter)` which does the Cookies.set.
`clearFilterCookie()` which does the Cookies.remove.